includes/class-boraboraio-page-loaded.php

```diff
@@ -38,35 +38,35 @@ function boraboraio_execute_on_load_page_hook_event(): void
 
     // Check if user session exists and is valid
     $userId = get_current_user_id();
+
+    // user not logged in - redirect to login page
+    $redirect_no_auth_id = carbon_get_theme_option(Boraboraio_Setting::BORA_BORA_IO_REDIRECT_NO_AUTH)[0]['id'] ?? 0;
+    $redirect_no_auth_url = esc_url(get_permalink($redirect_no_auth_id));
+    if ($userId === 0) {
+        wp_redirect($redirect_no_auth_url);
+        exit;
+    }
+
     $userSession = $sessionManager->getUserSession($userId);
 
     // If the session does not exist or is invalid, reload the information from the Bora Bora API
     $redirect_no_auth_id = carbon_get_theme_option(Boraboraio_Setting::BORA_BORA_IO_REDIRECT_NO_AUTH)[0]['id'] ?? 0;
```

Comment:
brauchen wir die zeile noch? wird das nicht schon in 43 gemacht?

Reply:
was genau meinst du?

Zeile 43:
`    $redirect_no_auth_id = carbon_get_theme_option(Boraboraio_Setting::BORA_BORA_IO_REDIRECT_NO_AUTH)[0]['id'] ?? 0;
`